expense/src/workflows/expense.ts

```diff
@@ -1,7 +1,12 @@
-import { ExpenseStatus } from '../interfaces/workflows';
-import { createExpense } from '@activities/createExpense';
-import { payment } from '@activities/payment' ;
+import { Expense, ExpenseStatus } from '../interfaces/workflows';
 import { Trigger, sleep } from '@temporalio/workflow';
+import { Context } from '@temporalio/workflow';
+import * as activities from '../activities';
```

Comment:
Same here and in the rest of the samples:
```suggestion
import type * as activities from '../activities';
```